scripts/TweakPSet.py

```diff
@@ -189,6 +188,13 @@ def createScriptLines(opts, pklIn):
         tweak.addParameter("process.source.lastEvent",
                            "customTypeCms.untracked.uint32(%s)" % opts.lastEvent)
 
+    if not inputFile["lfn"].startswith("MCFakeFile"):
+        # see https://github.com/dmwm/CRABServer/issues/7349
+        tweak.addParameter("process.source.skipEvents", "customTypeCms.untracked.uint32(0)")
+        print("Since the user set config.JobType.pluginName = 'Analysis', then")
```

Comment:
is this printout really needed at this point ? I am sure it helped for intial testing and debugging, but for what goes in all stdout of all jobs one line on the tune of "skip events set to 0" should be enough.

Reply:
Should be better now